### ISSUE TITLE
Horizontal scrollbar is nonfunctional

### ISSUE THREAD

**Issue by musicsac** [dotnet/try#25]
As I create a lengthy line, I can see my text as I write off the page (with the window scrolling to the right as it should), but the start of the line cannot be viewed by moving the horizontal scrollbar as expected. Arrow keying my cursor to the left is a workaround.
Microsoft Edge 41.16299.15.0
Zoom at 150%

**Comment by jonsequitur** [comment]
Additional info: It looks like there's both an outer scrollbar and an inner scrollbar (see screen shot). The inner scrollbar is the editor's scrollbar and does work if you scroll it vertically into view, and the outer scrollbar is an IFRAME scrollbar which we should not be seeing.

![image](https://user-images.githubusercontent.com/547415/34060138-391c7294-e197-11e7-87bb-e8d2dc655e66.png)

**Comment by jonsequitur** [comment]
@jdanyow

**Comment by jdanyow** [comment]
ack

**Comment by TyOverby** [comment]
@jdanyow: Any update on this?

**Comment by jdanyow** [comment]
We have someone looking at this. Holidays, other priorities, etc.

https://mseng.visualstudio.com/CSI/_workitems/edit/1147585

**Comment by TyOverby** [comment]
@jdanyow: ok, great!  I'll close this one and keep an eye on your issue.  If you need any functionality from us, please don't hesitate to get in touch!

**Comment by jdanyow** [comment]
Will do, thanks!

**Comment by wibjorn** [comment]
@TyOverby 

Believe this issue to be external to Docs.

TL;DR: Set `overflow: hidden` on the html element inside the iframe.

After some research, I could only reproduce the problem in Edge, but not only on high zoom scenarios. The iframe generated begins on page load correctly with no scrollbars, but when typing something, for example "a" the autosuggest (div.monaco-scrollable-element>div.monaco-list-rows) is given a height greater than the height of the editor box, which is likely causing the scrollbars to appear.

That said, this can all be mitigated by setting a style of `overflow: hidden` on the html element inside the iframe. Let me know if there's anything else I can do to help!

**Comment by TyOverby** [comment]
@wibjorn: what happens when you add `overflow: hidden` to the element containing the iframe?

**Comment by wibjorn** [comment]
@TyOverby At least when I did it in the Edge dev tools, the extra scrollbars never appear on Edge, even when you let the autosuggest div propagate.

**Comment by wibjorn** [comment]
@TyOverby Excuse me, I misunderstood your question. If you set `overflow: hidden` on the element containing the iframe, it has no effect.

**Comment by TyOverby** [comment]
If I understand this bug correctly, it seems like there are two elements with scrollbars when the bug is present: 

* One element that docs.microsoft.com controls
* One element that try.dot.net controls

Is that accurate?

**Comment by wibjorn** [comment]
From what we saw, the scrollbars are inside the iframe, or resulting from the styles of the iframe. Try placing the overflow hidden on the html element in the iframe, this should fix the problem entirely.

**Comment by colombod** [comment]
@TheRealPiotrP this is fixed with the release of the fix for #57 , we should close this

**Comment by TheRealPiotrP** [comment]
Thanks for fixing this @colombod!
